data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/discovery/DiscoveryMode.java

```diff
@@ -13,7 +13,8 @@
 public enum DiscoveryMode {
     STATIC(StaticPeerListProvider::createPeerListProvider),
     DNS(DnsPeerListProvider::createPeerListProvider),
-    AWS_CLOUD_MAP(AwsCloudMapPeerListProvider::createPeerListProvider);
+    AWS_CLOUD_MAP(AwsCloudMapPeerListProvider::createPeerListProvider),
+    LOCAL(LocalPeerListProvider::createPeerListProvider);
```

Comment:
I think it might make sense to name this `SINGLE_NODE` and also `SingleNodePeerListProvider`. Thoughts?

Reply:
IMO local is fine. SingleNode is not very indicative of local instance/node unless I misunderstand this DiscoveryMode

Reply:
`SINGLE_NODE` can be a confusing customer experience as they can define single node even in static list. Maybe a `LOCAL_NODE`?

Reply:
I'm fine with `LOCAL_NODE` as well. And we can change it any time prior to the release if we want.

As a point of reference, OpenSearch uses the "single node" concept: `discovery.type=single-node`